### ISSUE TITLE
Snapfit won't Snap

### ISSUE THREAD

**Issue by MxRemy** [madaeon01/LumiBee_beta#6]
Segment A won't snap closed closed on Segment B. The part shown in the red box::
![Won't Snap](https://user-images.githubusercontent.com/6307886/195212203-f5beff23-5eaf-4902-9228-dd40708e34a8.png)

I think it's because my countersunk screws (in the blue circles) won't actually sit flush against the snapfit connectors. As evidence of this, Segment B has no trouble snapping closed on Segment C...

Actually, all the countersunk screws protrude a little bit, from every part that uses them. I think the inset area needs to be a teeny bit bigger? Or else it might just be my screws, they were cheap. Not sure how variable the countersinking is on screws.

**Comment by madaeon01** [comment]
You can simply use a cutter, place it at an angle, and do a couple of rotations to increase the countersunk area.

**Comment by MxRemy** [comment]
Good call! I'll do that, but when I get a chance, should I try and make the countersunk area deeper/wider in the file itself? Or do you think this is just happening with my copies?
